### PR TITLE
feat: add get and set name commands to project

### PR DESCRIPTION
**Pull request by LiamConnors**
https://github.com/prefix-dev/pixi/issues/1359

### CODE REVIEW

File: src/cli/project/name/mod.rs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+use crate::Project;
+use clap::Parser;
+use std::path::PathBuf;
+
+/// Commands to manage project name.
+#[derive(Parser, Debug)]
+pub struct Args {
+    /// The path to `pixi.toml` or `pyproject.toml`
+    #[clap(long, global = true)]
+    pub manifest_path: Option<PathBuf>,
```
Comment (ruben-arts) [inline src/cli/project/name/mod.rs:13]: We just switched to use the `ProjectConfig` struct for this as much as possible so we only have on place to change the manifest_path. Check out the project/channel where I changed that a few days ago.

**Review by ruben-arts — CHANGES_REQUESTED** [review]
Thank you! Looks great, have one small comment.

**Review by ruben-arts — CHANGES_REQUESTED** [review]
Thank you! Looks great, have one small comment.

File: src/cli/project/name/mod.rs (hunk 1 above)
Comment (LiamConnors) [reply src/cli/project/name/mod.rs:13]: Thanks! Taking a look.

Comment (LiamConnors) [reply src/cli/project/name/mod.rs:13]: Thanks for your feedback @ruben-arts. I'm looking at that PR and was wondering, why does the declaration of `project` move out of the `execute` function in `mod.rs`? https://github.com/prefix-dev/pixi/pull/2569/files#diff-531dc39e5f51cb4dc5acf88fe6aa4e25f90a6d492724f3f8668d12f9f6cce9cbL107-R106

Can it stay and be passed to the `execute` functions on `get` and `set`?

```
let project = Project::load_or_else_discover(args.project_config.manifest_path.as_deref())?;

match args.command {
    Command::Get => get::execute(project).await?,
    Command::Set(args) => set::execute(project, args).await?,
}
```

Comment (ruben-arts) [reply src/cli/project/name/mod.rs:13]: Good question! In this case we don't need to do something with the configuration, so we don't need to do it in the sub commands themselves. So it's fine as you build it already!

**Comment by ruben-arts** [comment]
While testing it out, I added a small test. For the rest, this looks good to me! Assuming it was ready for review already.

**Review by ruben-arts — APPROVED** [review]

**Comment by LiamConnors** [comment]
Thanks @ruben-arts!

**Pull request merged**